nion/swift/Inspector.py

```diff
@@ -292,6 +292,13 @@ def finish_widget_content(self) -> None:
         """Subclasses should all this after calls to add_widget_content."""
         pass
 
+    def remove_widget_from_content(self, widget: UserInterface.Widget) -> None:
+        # need to remove widget and spacing from before the widget
+        index = self.__section_content_column.children.index(widget)
+        spacing = self.__section_content_column.children[index - 1]
```

Comment:
Is this guaranteed to have a spacing before the widget?

Reply:
Yes based on the function `add_widget_to_content` - line 286. If I didn't do that, the space there grows every time you interact with any of the controls.

Reply:
This is unused. It can be removed.